fragments/rhn-register.sh

```diff
@@ -44,7 +85,7 @@ subscription-manager repos \
                      --enable="rhel-7-server-rpms" \
                      --enable="rhel-7-server-extras-rpms" \
                      --enable="rhel-7-server-optional-rpms" \
-                     --enable="rhel-7-server-ose-3.2-rpms"
+                     --enable="rhel-7-server-ose-${OSE_VERSION}-rpms"
```

Comment:
Tested shell behavior with "rhel-7-server-ose-$OSE_VERSION-rpms" and the shell replacement works correctly. Updating.